### ISSUE TITLE
Palette and typography enhancements

### ISSUE THREAD

**Issue by nicholasswanson** [ResistanceCalendar/design#9]
Just doing an audit to ensure consistency and usability.

**Comment by nicholasswanson** [comment]
**Typography**
Recommend standardizing to [Roboto,](https://fonts.google.com/specimen/Roboto) a free and flexible webfont.

Can implement from Google Fonts by adding to `head`: `<link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,700,700i" rel="stylesheet">`

and calling `font-family: 'Roboto', sans-serif;` in CSS

**Comment by nicholasswanson** [comment]
**Palette**
Minor modifications, including pink to slightly more red, adding standardized grays, tweaking green.

<img src="https://cloud.githubusercontent.com/assets/3856906/23637840/c4d11ba8-0292-11e7-87e7-051a9d536261.png">

gray-xdark: #0D0B13
brand-red: #EE3656
brand-green: #23B09A
gray-medium: #6B6A73
gray-light: #D2D3DC
white: #ffffff

**Comment by dentemple** [comment]
Sorry, I have deuteranomaly.  Can you list the color codes in the same order as the picture, please?

The colors look great, and at my level of CVD they're nice and distinctive to each other.  They're certainly very pleasant to look at.

**Comment by nicholasswanson** [comment]
@dentemple done. Great to have feedback from a color blindness perspective.

**Comment by InciteDemocracy** [comment]
Yes, let's improve the colors! If we go too far toward red-red we end up looking like a BLACK/RED propaganda poster. We may want to consider something halfway, like this second proposal: 
![screenshot 2017-03-08 07 51 25](https://cloud.githubusercontent.com/assets/25624431/23716461/7ed41694-03d4-11e7-93f3-687001017f86.png)

gray-xdark: #0D0B13
brand-pink: #f42966
brand-green: #23B09A
gray-medium: #6B6A73
gray-light: #D2D3DC
white: #ffffff

**Comment by InciteDemocracy** [comment]
The red (PROP 1) is growing on me. What if we add an off-white/light-yellow to the pallet like this:

![screenshot 2017-03-08 11 56 15](https://cloud.githubusercontent.com/assets/25624431/23725872/50b2fe98-03f6-11e7-988a-074af3ff0635.png)

**Comment by nicholasswanson** [comment]
@InciteDemocracy I would lean toward choosing either a gray OR brown (yellow) palette. So if you are feeling strongly about the lighter yellow, I'd want to adjust the grays to be more on the brown scale to fit.

Thoughts on below?

<img width="600" src="https://cloud.githubusercontent.com/assets/3856906/23735543/0e26371c-043b-11e7-91c1-d2063ad0bf7b.png">

**Comment by bripenney** [comment]
Personally I think the pink and green are still too intense on the eyes. There also isn't really enough contrast between the pink, green and gray right now. 
<img width="589" alt="screen shot 2017-03-09 at 10 43 52 pm copy" src="https://cloud.githubusercontent.com/assets/24579735/23782416/f9742cb8-0519-11e7-9964-a1d74308bc9e.png">

Between brown and gray, I definitely would want to go with gray. 

This is more what I would be thinking. It's a bit friendlier, but you never really know how well a color palette is going to work until you start building out some pages. Also I know we're trying to be resistance, but I don't think a color palette that's any softer takes away from that. We'll still use bold type, layouts, and this is still a pretty vibrant palette particularly with the pink.

<img width="1206" alt="screen shot 2017-03-09 at 10 47 58 pm" src="https://cloud.githubusercontent.com/assets/24579735/23782495/77568180-051a-11e7-8fcf-62dd039653e7.png">

**Comment by bripenney** [comment]
Yeah I'm already seeing that that pink I just made won't work in designing out some things. We'll work on it!

**Comment by bripenney** [comment]
As far as typography goes, Roboto is great. Open Sans would also be a good option.

**Comment by bripenney** [comment]
![resistance_colors](https://cloud.githubusercontent.com/assets/24579735/23837087/acd94e7c-0750-11e7-9fa8-d3a7961df849.png)
